src/wp-admin/themes.php

```diff
@@ -307,9 +307,17 @@
 $current_theme_actions = array();
 
 if ( is_array( $submenu ) && isset( $submenu['themes.php'] ) ) {
+	$forbidden_paths = array(
+		'themes.php',
+		'theme-editor.php',
+		'site-editor.php',
+		'edit.php?post_type=wp_navigation',
+	);
+
 	foreach ( (array) $submenu['themes.php'] as $item ) {
 		$class = '';
-		if ( 'themes.php' === $item[2] || 'theme-editor.php' === $item[2] || 0 === strpos( $item[2], 'customize.php' ) ) {
+
+		if ( in_array( $item[2], $forbidden_paths, true ) || 0 === strpos( $item[2], 'customize.php' ) ) {
```

Comment:
```suggestion
		if ( in_array( $item[2], $forbidden_paths, true ) || str_starts_with( $item[2], 'customize.php' ) ) {
```
PHP 8 introduced `str_starts_with()` and we have polyfilled this in 5.9.

Reply:
Thank you for the review.
I've committed the suggested change.